### PR TITLE
Fix for PLTFRS-4878

### PR DESCRIPTION
**Pull request by phtrivier**
https://issues.sierrawireless.com/browse/PLTFRS-4878

GPS and Network locations should now be reread once every minute.
(Hopefully this does not drain the battery.)

### CODE REVIEW

**Comment by msangoi** [comment]
:+1:

File: src/com/sierrawireless/avphone/service/MonitoringService.java [hunk 1]
```diff
@@ -71,6 +73,12 @@
 
     private CustomDataSource customDataSource;
 
+    // FIXME(pht) for testing, to compare with "last known location"
```
Comment (sebz) [inline src/com/sierrawireless/avphone/service/MonitoringService.java]: This is how it all begins :stuck_out_tongue:

**Comment by KINFOO** [comment]
I just rebased on `master`.

@phtrivier You though this could not be merged as it not tested?
How about merge it, provide _rbejaran_ a beta to test?

**Comment by phtrivier** [comment]
@KINFOO the problem is I'm not sure of the impact on the battery, and I don't know how to test that before relasing it (Ideally, raul could test it, yes. I think I sent him an apk back in the days, but I don't know if he had time to check.)

**Comment by KINFOO** [comment]
@phtrivier 
Roger

**Comment by KINFOO** [comment]
I just push all buttons with this PR running on my phone.

I worth beta testing

**Pull request merged**